vendor/vendor.json

```diff
@@ -2,17 +2,91 @@
 	"comment": "",
 	"ignore": "test",
 	"package": [
+		{
+			"path": "appengine",
+			"revision": ""
+		},
+		{
+			"path": "appengine/urlfetch",
+			"revision": ""
+		},
+		{
+			"path": "appengine/user",
+			"revision": ""
+		},
+		{
+			"path": "appengine_internal",
+			"revision": ""
+		},
+		{
+			"path": "appengine_internal/base",
+			"revision": ""
```

Comment:
What are these packages without more details about repo?

Please make sure that all added packages in vendor are used?

Once added, they will forever located in repo even if we remove them in a new commit.

Reply:
All of these are added one by one by myself, all are used.